spec/fixtures/hiera/hiera.yaml

```diff
@@ -1,8 +1,10 @@
 ---
-:backends:
-  - yaml
-:yaml:
-  :datadir: 'spec/fixtures/hiera'
-:hierarchy:
-  - fqdn/%{fqdn}
-  - parameter_tests/%{parameter_tests}
+version: 5
```

Comment:
I thought that the spec tests still had to use the old version. Very cool if this now works!

Reply:
wooo! party on!